doc/kci_rootfs.md

```diff
@@ -0,0 +1,156 @@
+7. Now its time to re-build existing rootfs image using `kci_rootfs build`. It takes three arguments,
+`--config` refers to rootfs name which we want to build. `--data-path` points to debos files location.
+`--arch` refers to CPU arch we want to build. For example, to build a buster rootfs image for i386, 
+we can run
+
+    ```
+    ./kci_rootfs build --config buster --data-path jenkins/debian/debos --arch i386
+
+    ```
+   depending on your network speed, this will take sometime to complete. 
+
+8. If build is successful you should see message like
+
+    ```
+    cd ${ROOTDIR} ; find -H  |  cpio -H newc -v -o | gzip -c - > ${ARTIFACTDIR}/buster/i386/rootfs.cpio.gz | ./build_info.json
+    cd ${ROOTDIR} ; find -H  |  cpio -H newc -v -o | gzip -c - > ${ARTIFACTDIR}/buster/i386/rootfs.cpio.gz | 79539 blocks
+    Powering off.
+    ==== Recipe done ====
+    ```
+    Finally newly built rootfs images can be found at `jenkins/debian/debos/buster/i386/`
+
+    ```
+    $ ls jenkins/debian/debos/buster/i386/
+    build_info.json  full.rootfs.cpio.gz  full.rootfs.tar.xz  initrd.cpio.gz  rootfs.cpio.gz  rootfs.ext4.xz
+    ```
+
+### Create a new rootfs image 
+
+Now we know how to build default kci_rootfs images. Lets look at how to add simple `stretch` rootfs image. 
+
+1. First we need to add appropriate entries to `rootfs_config.yml` file.
+
+    ```
+      stretch:
+        rootfs_type: debos
+        debian_release: stretch
+        arch_list:
+          - amd64
+          - arm64
+        extra_packages_remove:
+          - e2fslibs
+          - e2fsprogs
+    ```
+
+  Above entry will create rootfs named `stretch` for CPU amd64 and arm64 architectures without `e2fslibs` and  `e2fsprogs` packages. List of possible `rootfs-config` yaml entries and its description are listed below:
+
+  | Entry                 | Description |
+  | ----------------------| ----------- |
+  | stretch               | Unique rootfs configuration name. |
+  | rootfs_type           | Tool used for rootfs creation. |
+  | debian_release        | Desired Debian OS version. |
+  | arch_list             | Desired list of CPU architecture. |
+  | extra_packages_remove | Specifies list of packages to remove from rootfs. |
+  | script                | Custom script to be executed during rootfs image creation. |
+  | test_overlay          | Create a directory layout on final rootfs image as provided. |
+  | extra_packages        | Installs specified packages on rootfs image. |
+  | extra_files_remove    | Removes specified files from rootfs image. |
+    
+
+2. Now validate `rootfs-config.yml` entries using `./kci_rootfs validate` and verify that it didn't report any errors.
+
+3. If no issues reported during validation, start the build process using, 
+
+    ```
+    ./kci_rootfs build --config stretch --data-path jenkins/debian/debos --arch amd64
+    ```
+    and wait for its completion and check `jenkins/debian/debos/stretch/amd64/` for newly built stretch rootfs image.
+
```

Comment:
Maybe we could put here a line like "if everything went find you should see something like that"?

Reply:
typo s/find/fine/